### PR TITLE
Rename ulid to ubid

### PR DESCRIPTION
**Pull request by perryr16**
#### Any background context you want to provide?
Currently, a property state and a tax lot have UBID and ULID (respectively) columns. The columns represent a previous understanding that a UBID was to be globally unique and that a building would only contain one. However, with the tools developed by PNNL, the probabilistic nature of UBID can be leveraged more fully with SEED.

#### What's this PR do?
TaxLots now have UBIDs, not ULIDs. This PR renames or removes uses of ULID

#### What are the relevant tickets?
#4029

### CODE REVIEW

**Comment by perryr16** [comment]
Because this PR is just a find replace/delete I didn't want it to clutter future work. Also thinking of merging features into a non-develop branch (`ubid-model-working-branch`) until it's ready.

File: seed/tests/test_ubid_views.py [hunk 1]
```diff
@@ -211,12 +208,9 @@ def test_decode_ulid_results_returns_a_summary_dictionary(self):
...
         expectation = {
-            "ubid_unpopulated": 0,
-            "ubid_successfully_decoded": 0,
-            "ubid_not_decoded": 0,
-            "ulid_unpopulated": 1,
-            "ulid_successfully_decoded": 1,
-            "ulid_not_decoded": 1
+            "ubid_unpopulated": 1,
+            "ubid_successfully_decoded": 1,
+            "ubid_not_decoded": 1,
```
Comment (haneslinger) [inline seed/tests/test_ubid_views.py:213]: why did the numbers change?

**Review by haneslinger — APPROVED** [review]
clean work

**Comment by axelstudios** [comment]
This work has been merged as part of #4074

**Pull request closed without merging**